examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<blockquote>&nbsp;(find detailed information&nbsp;<a href="https://pennylane.readthedocs.io/en/stable/code/api/pennylane.qaoa.cost.maxcut.html" target="_blank">here</a>)</blockquote><p>See this page to learn more about the QAOA module.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>